010_experiments/010_experiments.tex

```diff
@@ -382,4 +382,25 @@ \subsection{Conclusion}
 \label{subsec: E2-Conclusion}
 The change in reshuffle period and number of ticks per floor does not have a significant impact on the ability of the agents to self-organise. However, increasing the reshuffle period does increase the number of deaths and so would make reaching stability more difficult.
 
+\section{Experiment 3}
+\label{sec: Experiment 3}
+\subsection{Aims}
+\label{subsec: E3-Aims}
+The aim of this experiment is to analyse the average distribution of food on each floor of a 100 floor tower.
+The simulations will have a duration of 60 days and an initial food of 800. Two simulations are proposed to assess this: a tower consisting entirely of \texttt{randomAgent} and a tower consisting entirely of custom agents from the different teams, 1 to 7. The aim is to illustrate that the average food per floor decreases more slowly when the custom agents are assessed, showing that a sufficient level of self-organisation has been achieved to more effectively distribute food.
```

Comment:
```suggestion
The simulations will have a duration of 60 days and an initial food of 800. Two simulations are proposed to assess this: a tower consisting entirely of \texttt{randomAgent} and a tower consisting entirely of custom agents from the different teams, 2 to 7. The aim is to illustrate that the average food per floor decreases more slowly when the custom agents are assessed, showing that a sufficient level of self-organisation has been achieved to more effectively distribute food.
```
Not sure about this but is team 1s considered to be a custom agent? Its an amazing selfish agent tho

Reply:
ah good catch, thanks team 1 is definitely not custom

Reply:
lovelyyy